From fef1cc5ba1e505d1b9f33f38e64a585b836b5b2e Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:20:40 -0400
Subject: [alsa] Export snd_pcm_constraint_mask64()

Message-id: <140972823573-133-patchset-jkysela@redhat.com>
Patchwork-id: 90833
O-Subject: [RHEL 7.1 PATCH 132/442] ALSA: Export snd_pcm_constraint_mask64()
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Allow modules to use it, fixing a build failure when the newly added
ADAU1977 driver is built as a module.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Mark Brown <broonie@linaro.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 63a5d4c6a71fda7ac567627d6ac4a74e9555e97c)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/core/pcm_lib.c b/sound/core/pcm_lib.c
index 6630a4e..ce83def 100644
--- a/sound/core/pcm_lib.c
+++ b/sound/core/pcm_lib.c
@@ -1240,6 +1240,7 @@ int snd_pcm_hw_constraint_mask64(struct snd_pcm_runtime *runtime, snd_pcm_hw_par
   return -EINVAL;
  return 0;
 }
+EXPORT_SYMBOL(snd_pcm_hw_constraint_mask64);
 
 /**
  * snd_pcm_hw_constraint_integer - apply an integer constraint to an interval
-- 
1.7.1